src/Discord.Net.Rest/Entities/Interactions/RestApplicationCommandOption.cs

```diff
@@ -39,6 +39,11 @@ public class RestApplicationCommandOption : IApplicationCommandOption
         /// </summary>
         public IReadOnlyCollection<RestApplicationCommandOption> Options { get; private set; }
 
+        /// <summary>
+        /// The allowed channel types for this option.
+        /// </summary>
+        public IReadOnlyCollection<ChannelType>? ChannelTypes { get; private set; }
```

Comment:
I would remove the nullable type here just to follow along with the rest of the properties within the entity